src/constants.ts

```diff
@@ -19,8 +19,6 @@ export type ${HTML_STRING_NAME} = string`
 export const BASE_SYSTEM_FIELDS_DEFINITION = `// System fields
 export type BaseSystemFields<T = never> = {
 \tid: ${RECORD_ID_STRING_NAME}
-\tcreated: ${DATE_STRING_TYPE_NAME}
-\tupdated: ${DATE_STRING_TYPE_NAME}
```

Comment:
Is this now part of pb_schema.json?

Reply:
Exactly with 0.23.0 pocketbase will no longer add `updated` / `created` fields automatically. The UI will just add them for the user but they could be removed. This can be viewed in the [demo environment](https://pocketbase.io/demo/). And thats great this way it will automatically fix #92 ^^

The field definition would look like this:


```
      {
        "hidden": false,
        "id": "autodate2990389176",
        "name": "created",
        "onCreate": true,
        "onUpdate": false,
        "presentable": false,
        "system": false,
        "type": "autodate"
      },
      {
        "hidden": false,
        "id": "autodate3332085495",
        "name": "updated",
        "onCreate": true,
        "onUpdate": true,
        "presentable": false,
        "system": false,
        "type": "autodate"
      }
```

Reply:
Excellent!